pyroll/core/transport/hookimpls/roll_pass_entry.py

```diff
@@ -0,0 +1,37 @@
+from ..roll_pass_entry import RollPassEntry
+from ... import RollPass
+
+
+def check_if_successor_is_roll_pass(self):
+    if not isinstance(self.next, RollPass):
+        raise ValueError("Unit can only follow a RollPass unit.")
+
+
+@RollPassEntry.length
+def length(self: RollPassEntry):
+    check_if_successor_is_roll_pass(self)
+    return self.prev.exit_point + self.prev.roll.nominal_radius
```

Comment:
The same as below for exit. But here I think there is an dependence on entry point. Can be cumbersome because we need to access the next unit.

Reply:
Forgot to alter that code is relict from c + p. Will change

Reply:
Accessing next unit was tried in #171. Needs a fallback for first run, but otherwise it is fine.